.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug Report
+about: Bug report related to the technical aspects of the project.
+title: Description of the issue
+labels: 'bug'
+assignees: ''
+---
+
+Per Mattermost guidelines, GitHub issues are for bug reports: <http://www.mattermost.org/filing-issues/>.
+
+For troubleshooting see: http://forum.mattermost.org/.
+For feature proposals see: http://www.mattermost.org/feature-requests/
+
+If you've found a bug--something appears unintentional--please follow these steps:
+
+1. Confirm you’re filing a new issue. [Search existing tickets in Jira](https://mattermost.atlassian.net/jira/software/c/projects/MM/issues/) to ensure that the ticket does not already exist.
```

Comment:
Is that correct for all repos? E.g. Focalboard uses Focalboard for bugs and plugins use GitHub.